UserInterface.adoc

```diff
@@ -80,52 +128,368 @@ fundamental A: If it does behave that way now, I'd think it would be wise to
 make it not behave that way in the future.
 ////
 
-=== Navigation Panel
-
-The left side of the screen can be used to change what view the content panel is
-actively presenting.
-The side panel is used to navigate between different parts of the user
-interface.
-From here you can change the view in the main panel to:
-
-* Part settings
-* Part Grid - the active part id
-* Browser
-* Mixer
-* Kits
-* Kit Grid - the active kit id
-* Macro Learn
-* Effects
-* Add synth
-* Add synth Voice Grid - the active voice id
-* Sub synth
-* Pad synth
-
-For all of the grid selectors you can change the active id with the left mouse
-button or you can enable/disable the selected ID via the middle mouse button.
+==== Navigation Panel
+
+The navigation panel is situated on the left side of the Zyn-Fusion UI and, much like the
+status panel, is always present there.
+
+It is used to change what the main panel is actively presenting.
+The options, from top to bottom, include:
+
+* PART SETTINGS
+* Part grid
+* BROWSER
+* MIXER
+* KITS
+* Kit grid
+* MACRO LEARN
+* EFFECTS
+* The ADDSynth panel
+* The ADDSynth voice grid
+* The SUBSynth panel
+* The PADSynth panel
+
+Each grid houses multiple elements with the same functionality,
+and has one lit-up square, which is the currently displayed element.
+Each active element has a border around it.
+*To view an element in the grid, left-mouse click it.*
+*To enable an element in the grid, middle-mouse click it.*
+
+// TODO: Capitalize each word?
+==== Main Panel (Introduction)
+
+The main panel is where most of the action happens.
+
+The scope of that is, understandably, very large, so it warrants
+<<_main_panel, its own section in this part of the manual, visible below>>.
+
+==== Virtual keyboard panel
+
+The virtual keyboard is located at the bottom of the Zyn-Fusion interface.
+
+// TODO
+
+Modulation wheel::
+    Desc
+
+Keyboard::
+    Desc
+
+VELOCITY::
+    Desc
+
+VRND::
+    Desc
+
+OCTAVE::
+    Desc
+
+C.VAL::
+    Desc
+
+MIDI CC dropdown menu::
+    Desc
 
 === Main Panel
 
-Here's where the most of the action happens.
-When you start Zyn-Fusion it'll begin by showing you the Browser.
-Use the Side Bar to select what should be displayed here.
+The main panel covers all the core functionality in Zyn-Fusion,
+in regard to sound design, effects, mixing, and so on.
+
+We'll go over each subpanel from top to bottom as seen in the <<_navigation_panel, navigation panel>>.
+
+==== Part settings
+
+// TODO: Image with colors, or multiple images
+
+This subpanel is dedicated to setting up each the properties of each part.
+
+.What is a part?
+NOTE: A part is a single element in ZynAddSubFX capable of generating sound.
+In one part, any of ZynAddSubFX's three synths can be active.
```

Comment:
This blurb right here is what I was hoping for in a user manual, something to introduce the concepts without hanging on the minutia.